lib/config/index.js

```diff
@@ -987,9 +987,9 @@ function getHostnameSafe(gcpId = null) {
   try {
     // Check for any special hostname scenarios.
     // If not applicable, use the os.hostname()
-    if (config.heroku.use_dyno_names && process.env.DYNO) {
+    if (config.heroku?.use_dyno_names && process.env.DYNO) {
       _hostname = process.env.DYNO || os.hostname()
-    } else if (config.gcp_cloud_run.use_instance_as_host && process.env.K_SERVICE && gcpId) {
+    } else if (config.utilization?.gcp_use_instance_as_host && process.env.K_SERVICE && gcpId) {
```

Comment:
We shouldn't have empty configuration objects. There isn't a need for the `?`:

```suggestion
    } else if (config.utilization.gcp_use_instance_as_host && process.env.K_SERVICE && gcpId) {
```

Reply:
Yeah, this was a bandaid because one of the versioned tests set `config.utilization = null`. I'll just change it to `{}`.